src/views/ingresses/details/tabs/details/components/IngressDetailsSection/IngressDetailsSection.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC } from 'react';
+
+import { IngressModel, NamespaceModel } from '@kubevirt-ui/kubevirt-api/console';
+import { IoK8sApiNetworkingV1Ingress } from '@kubevirt-ui/kubevirt-api/kubernetes/models';
+import {
+  getGroupVersionKindForModel,
+  ResourceLink,
+  Timestamp,
+  useAccessReview,
+  useAnnotationsModal,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { DetailsItem } from '@utils/components/DetailsItem/DetailsItem';
+import { LabelList } from '@utils/components/DetailsItem/LabelList';
+import { OwnerReferences } from '@utils/components/OwnerReference/owner-references';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+import {
+  getCreationTimestamp,
+  getLabels,
+  getName,
+  getNamespace,
+  getUID,
+} from '@utils/resources/shared';
+
+import TLSCert from './TLSCert';
+
+type IngressDetailsSectionProps = {
+  ingress: IoK8sApiNetworkingV1Ingress;
+};
+
+const IngressDetailsSection: FC<IngressDetailsSectionProps> = ({ ingress }) => {
+  const { t } = useNetworkingTranslation();
+  const annotationsModalLauncher = useAnnotationsModal(ingress);
+
+  const [canUpdate] = useAccessReview({
+    group: IngressModel?.apiGroup,
+    name: getName(ingress),
+    namespace: getNamespace(ingress),
+    resource: IngressModel?.plural,
+    verb: 'patch',
+  });
+
+  return (
+    <>
+      <DetailsItem label={t('Name')} obj={ingress} path={'metadata.name'} />
+      {getNamespace(ingress) && (
+        <DetailsItem label={t('Namespace')} obj={ingress} path="metadata.namespace">
+          <ResourceLink
+            groupVersionKind={getGroupVersionKindForModel(NamespaceModel)}
+            name={getName(ingress)}
+            namespace={null}
+            title={getUID(ingress)}
+          />
+        </DetailsItem>
+      )}
+      <DetailsItem
+        canEdit={canUpdate}
+        editAsGroup
+        label={t('Labels')}
+        obj={ingress}
+        onEdit={annotationsModalLauncher}
+        path="metadata.labels"
+        valueClassName="details-item__value--labels"
+      >
+        <LabelList
+          groupVersionKind={getGroupVersionKindForModel(IngressModel)}
+          labels={getLabels(ingress)}
+        />
+      </DetailsItem>
+      <dt>{t('TLS certificate')}</dt>
```

Comment:
shouldnt this also be DetailsItem ?

Reply:
Yes, I'm not sure how I missed that one. Fixed. Thanks!